tests/test_document.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import unicode_literals
+
+import io
+import tempfile
+
+from nose import tools
+
+try:
+    from pyPdf2 import PdfFileReader
```

Comment:
It's the capitalization, it should be "PyPDF2" :)

https://github.com/mstamy2/PyPDF2/blob/master/PyPDF2/__init__.py